docs/getting-started-with-cosmosdb.md

```diff
@@ -0,0 +1,53 @@
+# Getting Started with Scalar DB on Cosmos DB
+
+## Overview
+This document briefly explains how you can get started with Scalar DB on Cosmos DB with a simple electronic money application.
+
+## Install prerequisites
+
+Scalar DB is written in Java. So the following software is required to run it.
+
+* [Oracle JDK 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) (OpenJDK 8) or higher
+* Other libraries used from the above are automatically installed through gradle
+
+## Cosmos DB setup
+You also need to configure a Cosmos DB account to get started with Scalar DB on Cosmos DB.
```

Comment:
I think that you create a Cosmos DB account and configure that in this section.
```suggestion
You also need to set up a Cosmos DB account to get started with Scalar DB on Cosmos DB.
```

Reply:
Thank you for the suggestion.
Document updated with the suggestion.